pages/measurement-sets/[id].js

```diff
@@ -266,6 +226,18 @@ export default function MeasurementSet({
               sequencingPlatforms={sequencingPlatforms}
             />
           )}
+          {controlFileSets?.length > 0 && (
+            <>
+              <DataAreaTitle>Control File Sets</DataAreaTitle>
+              <FileSetTable fileSets={controlFileSets} />
+            </>
+          )}
+          {relatedFileSet?.length > 0 && (
```

Comment:
`relatedFileSet` is required, so you don’t need the question mark. Rename to `relatedFileSets` since it’s an array.

Reply:
done